internal/cloudapi/server.go

```diff
@@ -78,7 +78,7 @@ func (b binder) Bind(i interface{}, ctx echo.Context) error {
 
 	err := json.NewDecoder(ctx.Request().Body).Decode(i)
 	if err != nil {
-		return echo.NewHTTPError(http.StatusBadRequest, fmt.Sprintf("Cannot parse request body: %v", err))
+		return echo.NewHTTPError(http.StatusBadRequest, "Cannot parse request body: %s", err)
```

Comment:
You should use `err.Error()` instead of `err` here, to make sure that the passed value type is `string`.